cherryontech/do-five-things
# feat: Add Settings page and base layout

## Context

Devs scaffolding the app :)

## Work done

- Added a "Settings" page to hack on.
- Created a `BaseLayout` component and automatically add it to all pages (we will want it later) 
- Cleaned up a bit of boilerplate styling

## Manual testing instructions

1. Yarn and bundle install and run db migrations
2. Serve app
3. Navigate to http://localhost:3000/inertia-example and then to http://localhost:3000/settings
- Expect that both pages have a header
- Expect that both pages have a footer

## Gotchas/Lessons Learned

Resources: 
- [Inertia rails](https://inertia-rails.netlify.app/guide/pages.html)
- [DFT Figma](https://www.figma.com/design/wBpAWs2jxyNCuI6u6mOYjw/DFT-APP?node-id=73-1492&t=uL0JG5Hgckoyne8q-0)
- [Example app with inertia, rails, and Vue3](https://github.com/ledermann/pingcrm)
